Tasks/KubernetesV1/src/kubernetescommand.ts

```diff
@@ -42,19 +42,40 @@ function getCommandOutputFormat(kubecommand: string) : string[] {
     return args;
 }
 
-function getCommandConfigurationFile() : string[] {
-    var args: string[] =[];
-    var useConfigurationFile : boolean  =  tl.getBoolInput("useConfigurationFile", false);
-    if(useConfigurationFile) {
-        var configurationPath = tl.getInput("configuration", true);
-        if(configurationPath && tl.exist(configurationPath))
-        {
+function getCommandConfigurationFile(): string[] {
+    var args: string[] = [];
+    var useConfigurationFile: boolean = tl.getBoolInput("useConfigurationFile", false);
+    let configurationPath = tl.getPathInput("configuration", false);
+    var inlineConfiguration = tl.getInput("inline", false);
+
+    if (!tl.filePathSupplied("configurationPath")) {
+        configurationPath = null;
+    }
+
+    if (useConfigurationFile && configurationPath == null && inlineConfiguration == null) {
```

Comment:
You don't need `==null` use `!!configurationPath` instead or just `configurationPath` 
`null` equals `false` in javascript conditions,